google/cloud/aiplatform/metadata/__init__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from google.cloud.aiplatform.metadata.metadata_store import _MetadataStore
```

Comment:
lint complaints about this not being used. But without this, UTs fail... there might be something I'm missing
@sasha-gitg

Reply:
You have two options. I think 1 is preferable because exposing it here is not fulfilling a CUJ but we can change that later:

1. Keep this `__init__` file empty for now. And directly reference `_MetadataStore` ie:

```Python
from google.cloud.aiplatform.metadata import metadata_store
```

2. Change the line below to: 

```Python
__all__ = (_MetadataStore, )
```

Reply:
Perfect, went with number 1 then.
thanks @sasha-gitg